### PR TITLE
Views

### PR DESCRIPTION
**Pull request by yuvenalmash**
## Changes Summary
- Implement the design from the wireframes
  - [Wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md)

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1,31 +1,17 @@
...
+<%= render 'shared/user', user: @user %>
+<ul class="posts-list">
+  <% @user.posts.each do |post| %>
+    <li class="posts-list-li">
+      <%= render 'shared/post', post: post %>
+      <% if post.comments.count > 0 %>
+        <ul class="comments-list">
+          <% post.comments.each do |comment| %>
+            <%= render 'shared/comment', comment: comment %>
+          <% end %>
```
Comment (rloterh) [inline app/views/posts/index.html.erb]: - [x] Good job, views are rendering correctly! 👏 However, in this activity we want you to use the methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md), in this case `post.recent_comments` method (to render _5 most recent comments_), like the example below:

```ruby
#app/views/posts/index.html.erb` page

 <% post.recent_comments.each do |comment| %>
  <p><%= comment.author.name %>: <%= comment.text %></p> 
 <% end %>
```

**Comments of post(s):**
image

Let's ensure our code fulfils this requirement! 👨‍💻

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Yuvenal, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 
🏁 🌴  🏃‍♂️ 🌳 🌲 🏡 

## Highlights 👍
💠 Views are rendering correctly!
💠 ReadMe looks good!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

**You can also consider:**
- Generally, it's a good idea that the subheading text _Author1_ should be replaced with your full name in the ReadMe. Implementing the above change(s) will make your ReadMe stand out even better! 😉

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (yuvenalmash) [reply app/views/posts/index.html.erb]: Thanks, I must have missed it!

Comment (Abenezer-Tilahun) [reply app/views/posts/index.html.erb]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @yuvenalmash 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**